metadata-db-app/README.md

```diff
@@ -46,17 +46,14 @@
 
   {
       "concept-type": "service",
-      "native-id": "provider service id",
-      "concept-id": "S1-PROV1",
-      "provider-id": "PROV1",
+      "native-id": "mod10cm",
       "user-id": "user101",
-      "metadata": "xml here",
-      "format": "application/serf+xml",
+      "metadata": "service edn here",
+      "format": "application/edn",
```

Comment:
Question about that: do variable and service data need to support any non-JSON data types like dates or sets?